.github/actions/dump-disk-info/action.yaml

```diff
@@ -0,0 +1,16 @@
+name: Dump disk space info
+description: Help debug running out of disk space on github CI
+inputs:
+  platform:
+    description: "Platform name"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Collect disk info
+      uses: pyTooling/Actions/with-post-step@v0.4.5
```

Comment:
Is it okay to use this?

Reply:
If we auto-get disk space info at start and end, this may be interesting. However if we just need the boot time, maybe it is simpler to embed shell executions directly.

Reply:
I think that is useful. maybe add some comment about this (i.e. why the action exists rather than just inline).